README.md

```diff
@@ -49,21 +129,24 @@ questions, or want to talk about future plans, please join us on Discord
 or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions); the
 developers monitor both channels.
 
-> [!IMPORTANT]
-> Jujutsu is an **experimental version control system**. While Git compatibility
-> is stable, and most developers use it daily for all their needs, there may
-> still be work-in-progress features, suboptimal UX, and workflow gaps that make
-> it unusable for your particular use.
-
 ### News and Updates 📣
 
+- **Oct 2023**: Version 0.10.0 is released! Now includes a bundled merge and
+  diff editor for all platforms, "immutable revsets" to avoid accidentally
+  `edit`-ing the wrong revisions, and lots of polish.
```

Comment:
[No Action Needed] I think there will be too many of these if we have a news item for every release without removing old ones. So, I think we shouldn't plan on these "News" being "append-only"; we could have an item form one or two latest releases.

Reply:
Yes, I figured it would probably only be updated whenever there was "Exciting news" (e.g. gitoxide migration complete, new website, native backend) and then probably "extra" stuff (we gave a talk, read a blog, etc.)